packages/core/src/runtime/useEventBus.ts

```diff
@@ -0,0 +1,8 @@
+import { AbstractRuntime } from "./abstractRuntime.ts";
+import { useRuntime } from "./RuntimeContext.ts";
+
+export function useEventBus() {
+    const runtime = useRuntime() as AbstractRuntime;
```

Comment:
Instead of having a complete "unknown" context, i think it would be better to have an AbstractRuntime. This way you don'T have to cast everytime you use it in core, but when using it in a context where we know which abstraction we need, than you can do useRuntime() as Runtime